### DISCUSSION TITLE
[Feature Request] Implement "Closed as duplicate (#{issue})" tag

### DISCUSSION THREAD

**Discussion by e-t-l** [community/community#11925] [Projects and Issues]
### Description
1. When closing an issue, there should be an option to mark it, "Closed as duplicate," with a prompt to select an existing issue (i.e. the original-issue that the newly-closed issue had duplicated). 
2. When this option used, the issue is given a `Closed as Duplicate (#{issue})` tag, where `#{issue}` is the original-issue number. 
3. When a user clicks this tag, they are redirected to the original-issue.

### Purpose
1. This would make it very easy to see, directly from the /issues page, which issues are duplicates and thus are unlikely to contain useful info. 
2. From the /issues page, a user could see that an issue is marked as a duplicate, and by clicking the tag, quickly navigate to the original-issue _without having to first open the duplicate link._
3. As a discrete tag, users would be able to filter it out of their issue search results if they don't want duplicates cluttering up the page.
4. This would be very useful for keeping large repos organized, especially if many issues are opened by end-users, not devs.

**Comment by RoboMWM** [comment]
Besides a visual 'event' entry, it seems that 'marking' an issue as a duplicate does nothing else. Is it even possible to search for issues 'marked as duplicate' by this method?

**Reply by e-t-l** [reply]
This 'method' is a proposal, and as such does not currently exist. It could be implemented to allow for searching or whatever you like.

Speaking of which, there are at least  4 examples od potential uses in the original post, if you care to read it.

**Reply by MLG-SERBUR** [reply]
I guess there's a feature in github discussions and/or issues to restrict specific users from replying to a specific issue/discussion? I cannot reply with my same account (I can only guess that the user has blocked me, providing no room for misunderstanding which happened here): 
![image](https://github.com/user-attachments/assets/33d5e434-3337-4d94-bf57-68ec363105ab)

Anyways, yes, that was made known to me that this is a feature request later on. I kept the comment as I was referring to the existing feature being quite lacking. (I should've edited my comment to clarify that, but by the time I was discussing it with others, I forgot, and I'm currently unable to do so.)

As such, I am in support of this feature request.

This feature would be far more useful than the "Not planned" option that was added, which that language assumes most issues are feature requests than bug reports. (Saying "not planned" for an invalid or duplicated bug report sounds like "wontfix" instead.)

**Comment by mikeukeul** [comment]
“Il est temps que GitHub et GitHub Desktop franchissent le pas vers une interface universelle et multilingue. Les extensions de traduction ne suffisent pas : elles bricolent là où une solution native apporterait cohérence, fiabilité et inclusion. Dans un monde où l’IA traduit le sumérien, pourquoi nos outils modernes ne pourraient-ils pas traduire toutes les langues vivantes ? Arrêtons le cloisonnement des pensées et ouvrons le monde des possibles.”
